openverse-api/catalog/api/models/audio.py

```diff
@@ -0,0 +1,180 @@
+from django.contrib.postgres.fields import JSONField, ArrayField
+from django.db import models
+from uuslug import uuslug
+
+import catalog.api.controllers.search_controller as search_controller
+from catalog.api.models import OpenLedgerModel
+from catalog.api.models.media import (
+    AbstractAltFile,
+    AbstractMedia,
+    AbstractMediaReport,
+    AbstractDeletedMedia,
+    AbstractMatureMedia,
+    AbstractMediaList,
+)
+from catalog.api.models.mixins import (
+    IdentifierMixin,
+    MediaMixin,
+    FileMixin,
+)
+
+
+class AltAudioFile(AbstractAltFile):
+    def __init__(self, attrs):
+        self.bit_rate = attrs.get('bit_rate')
+        self.sample_rate = attrs.get('sample_rate')
+        super(AltAudioFile, self).__init__(attrs)
+
+    @property
+    def sample_rate_in_khz(self):
+        return self.sample_rate / 1e3
+
+    @property
+    def bit_rate_in_kbps(self):
+        return self.bit_rate / 1e3
+
+    def __str__(self):
+        br = self.bit_rate_in_kbps
+        sr = self.sample_rate_in_khz
+        return f'<AltAudioFile {br}kbps / {sr}kHz>'
+
+    def __repr__(self):
+        return str(self)
+
+class AudioSet(IdentifierMixin, MediaMixin, FileMixin, OpenLedgerModel):
+    """
+    This is an ordered collection of audio files, such as a podcast series or
+    an album. Not to be confused with AudioList which is a many-to-many
+    collection of audio files, like a playlist or favourites library.
+
+    The FileMixin inherited by this model refers not to audio but album art.
+    """
```

Comment:
Do we need the `filesize` of the album art for something? I don't see if it’s used for thumbnails in the case of images for example. Asking because then we could name the url of the album art as `cover_url` or `artwork_url` in that case.

Reply:
The filesize field is not currently being used anywhere. I was just trying to be consistent with the fields used in images, so that we can reuse code as much as possible, especially when using the same image proxy for artwork as image media.